src/movie-area.css

```diff
@@ -1,7 +1,20 @@
 .area {
-  background-image: linear-gradient( #8e2de2, #4a00e0);
+  background-color: white;
```

Comment:
The purple color looked a bit childish, we can figure out a nice background color later. Maybe we can use a color template or just use an inspirational reference.